metron-interface/metron-alerts/e2e/alert-details/alert-status/alert-details-status.e2e-spec.ts

```diff
@@ -101,7 +104,7 @@ describe('metron-alerts alert status', function() {
     page.closeDetailPane();
   });
 
-  it('should add comments for tree view', () => {
+  xit('should add comments for tree view', () => {
```

Comment:
Any particular reason to skip this test?

Reply:
@iraghumitra Same question as above.